highlight/highlight
# remove unauthorized viewing form

## Summary



We have a form that shows up when Highlight admins view a customer session. 

![image](https://user-images.githubusercontent.com/58678/233691106-07e1eaa1-62a3-40df-a1e8-77c46530ef86.png)


After looking at the code, the audit reason wasn't being recorded: https://github.com/highlight/highlight/blob/e25924cd997e95fc9ccb1d02d95f610fda162e95/frontend/src/pages/Player/components/UnauthorizedViewingForm/UnauthorizedViewingForm.tsx#L13-L44

and it's also incorrectly showing for requests in the Highlight workspace (where the projectId != 1), see https://github.com/highlight/highlight/issues/4989#issuecomment-1514890269

## How did you test this change?



* Login to the render preview as a Highlight admin
* Observe that viewing a [session](https://frontend-pr-5075.onrender.com/4817/sessions/GFawhvIvv4kNdL7gP11I5ZyrLuyo) where `project_id` != 1  does not show the unauthorized prompt.
 
## Are there any deployment considerations?



N/A
